src/main/java/org/everit/jira/timetracker/plugin/JiraTimetrackerTableWebAction.java

```diff
@@ -422,8 +449,8 @@ public ApplicationUser getUserPickerObject() {
   private boolean isRealWorklog(final EveritWorklog worklog) {
     boolean isRealWorklog = true;
     if (issuesRegex != null) {
-      for (Pattern issuePattern : issuesRegex) {
-        boolean issueMatches = issuePattern.matcher(worklog.getIssue()).matches();
+      for (final Pattern issuePattern : issuesRegex) {
```

Comment:
Remove all of the unnecessary final modifier from the local variables from all of the modified classes.